### ISSUE TITLE
~100x regression in db_basic_test time on PPC in Travis

### ISSUE THREAD

**Issue by pdillinger** [facebook/rocksdb#6653]
PPC builds frequently timing out.

Recent data from "platform_dependent" of https://travis-ci.org/github/facebook/rocksdb/builds/671595193?utm_source=github_status&utm_medium=notification

PPC timestamps:
$ date; ${CXX} --version
Mon Apr  6 12:14:11 UTC 2020
===== Running db_basic_test (Mon Apr  6 12:31:46 UTC 2020)
===== Running db_with_timestamp_basic_test (Mon Apr  6 12:59:17 UTC 2020)
In short, 17.5 minutes compiling, 27.5 minutes in db_basic_test, long times spread across many test cases.

ARM timestamps:
$ date; ${CXX} --version
Mon Apr  6 12:09:25 UTC 2020
===== Running db_basic_test (Mon Apr  6 12:23:48 UTC 2020)
===== Running db_with_timestamp_basic_test (Mon Apr  6 12:24:49 UTC 2020)
In short, 14+ minutes compiling, 1 minute in db_basic_test

And a couple weeks ago, db_basic_test took 20 seconds on PPC:
https://travis-ci.org/github/facebook/rocksdb/jobs/666445282?utm_medium=notification&utm_source=github_status

**Comment by adamretter** [comment]
@pdillinger I could run two comparisons on a PPC64le box here if you want to know if it's just Travis?

**Comment by pdillinger** [comment]
It appears to be different behavior from the Travis builder 😡. See #6655 

https://travis-ci.org/github/facebook/rocksdb/jobs/671840930?utm_medium=notification&utm_source=github_status vs. using essentially the same revision two weeks ago https://travis-ci.org/github/facebook/rocksdb/jobs/664565410?utm_medium=notification&utm_source=github_status

Possibly(?) related to sometimes disk quota issue

**Comment by pdillinger** [comment]
Email sent to Travis support

**Comment by adamretter** [comment]
@pdillinger Sometimes with Travis it is better to open an issue here - https://travis-ci.community/c/environments/multi-cpu-arch/96

**Comment by pdillinger** [comment]
> Email sent to Travis support

Reply was not useful, but...

It appears that my PR didn't have the intended effect, because it seems that Travis built a rebased version of my commit 😡  https://github.com/facebook/rocksdb/commit/06befa5bb7420c74cb1d359c08918f65783d2350 instead of https://github.com/facebook/rocksdb/commit/59852b5e717ca83f3ea516ac0bec50abd0565e03

Launched a branch build:

https://travis-ci.org/github/facebook/rocksdb/builds/672176972

**Comment by pdillinger** [comment]
> https://travis-ci.org/github/facebook/rocksdb/builds/672176972

[==========] 92 tests from 5 test cases ran. (2005962 ms total)

vs. baseline two weeks ago https://travis-ci.org/github/facebook/rocksdb/jobs/664565410

[==========] 92 tests from 5 test cases ran. (12911 ms total)

155x slower on the same code

**Comment by pdillinger** [comment]
> @pdillinger Sometimes with Travis it is better to open an issue here - https://travis-ci.community/c/environments/multi-cpu-arch/96

Added to an existing issue: https://travis-ci.community/t/disk-quota-exceeded-on-ppc64le/8006

**Comment by pdillinger** [comment]
Someone sees better experience on travis-ci.com (vs. .org): https://travis-ci.community/t/disk-quota-exceeded-on-ppc64le/8006/3

**Comment by pdillinger** [comment]
Update:

Old bad: [----------] 48 tests from ParallelIO/DBBasicTestWithParallelIO (131162 ms total)

New, seems to be OK: [----------] 48 tests from ParallelIO/DBBasicTestWithParallelIO (1909 ms total)

I suspect this issue was fixed with #7101 where the problem manifesting on MacOS as complete hang would greatly and unpredictably slow down other non-Linux.
